### PR TITLE
Feature/us204 class of title

### PR DESCRIPTION
**Pull request by russwillis**

### CODE REVIEW

File: service/server.py [hunk 1]
```diff
@@ -343,14 +343,21 @@ def _create_string_date_and_time(datetoconvert):
...
+    # TODO use real date - this is reliant on new functionality to check the daylist
+    last_entry_date = _create_string_date_and_time(datetime(3001, 2, 3, 4, 5, 6))
     issued_date = _create_string_date_only(datetime.now())
     if title.get('edition_date'):
         edition_date = _create_string_date_only(datetime.strptime(title.get('edition_date'), "%Y-%m-%d"))
     else:
         edition_date = "No date given"
+    class_of_title = title.get('class_of_title')
+    # need to check for caution title as we don't display Class of title for them
+    is_caution = 'true' if title.get('is_caution_title') is True else 'false'
```
Comment (lgonczar) [inline service/server.py]: It would be best to make `is_caution` a boolean variable

**Comment by lgonczar** [comment]
Reviewed. Looks good, left some comment.

File: service/server.py (hunk 1 above)
Comment (russwillis) [inline service/server.py]: Didn't think I could pass a boolean through the template but if I can I will :)

**Comment by lgonczar** [comment]
LGTM

File: service/server.py [hunk 2]
```diff
@@ -343,14 +343,21 @@ def _create_string_date_and_time(datetoconvert):
...
+    # TODO use real date - this is reliant on new functionality to check the daylist
+    last_entry_date = _create_string_date_and_time(datetime(3001, 2, 3, 4, 5, 6))
     issued_date = _create_string_date_only(datetime.now())
     if title.get('edition_date'):
         edition_date = _create_string_date_only(datetime.strptime(title.get('edition_date'), "%Y-%m-%d"))
     else:
         edition_date = "No date given"
+    class_of_title = title.get('class_of_title')
+    # need to check for caution title as we don't display Class of title for them
+    is_caution = True if title.get('is_caution_title') is True else False
```
Comment (nick-gravgaard) [inline service/server.py]: This should be

``` python
    is_caution = title.get('is_caution_title') is True
```

File: service/templates/full_title.html [hunk 3]
```diff
@@ -219,7 +222,7 @@ <h3 class="register-c">Charges register</h3>
             <h4{% if 'entry_date' in entry %} class="shunt"{% endif %}>{{ entry['sequence_number'] }}:</h4>
             <p>{{ entry['full_text'] }}</p>
           </div>
-        <div>
+        </div>
       {% endfor %}
```
Comment (nick-gravgaard) [inline service/templates/full_title.html:226]: Good catch!

**Comment by lgonczar** [comment]
Are there acceptance tests passing for this?

**Comment by russwillis** [comment]
Acceptance tests are passing.

**Comment by Pheonnexx** [comment]
LGTM

**Pull request merged**